include/deal.II/base/polynomials_pyramid.h

```diff
@@ -37,10 +39,10 @@ class ScalarLagrangePolynomialPyramid : public ScalarPolynomialsBase<dim>
 
   /*
    * Constructor taking the polynomial @p degree as input.
-   *
-   * @note Currently, only linear polynomials (degree=1) are implemented.
    */
-  ScalarLagrangePolynomialPyramid(const unsigned int degree);
+  ScalarLagrangePolynomialPyramid(const unsigned int            degree,
```

Comment:
I guess this is a public class? So we need to keep the old interface, which might call the new one.